cloud-sql/sqlserver/mssql/server.js

```diff
@@ -0,0 +1,197 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const express = require('express');
+const mssql = require('mssql')
+const bodyParser = require('body-parser');
+
+const app = express();
+app.set('view engine', 'pug');
+app.enable('trust proxy');
+
+// Automatically parse request body as form data.
+app.use(bodyParser.urlencoded({extended: false}));
+app.use(bodyParser.json());
+
+// Set Content-Type for all responses for these routes.
+app.use((req, res, next) => {
+  res.set('Content-Type', 'text/html');
+  next();
+});
+
+// Create a Winston logger that streams to Stackdriver Logging.
+const winston = require('winston');
+const {LoggingWinston} = require('@google-cloud/logging-winston');
+const loggingWinston = new LoggingWinston();
+const logger = winston.createLogger({
+  level: 'info',
+  transports: [new winston.transports.Console(), loggingWinston],
+});
+
+// [START cloud_sql_server_mssql_create]
+let pool;
+const createPool = async () => {
+  let config = {pool: {}};
+  config.user = process.env.DB_USER; // e.g. 'my-db-user'
+  config.password = process.env.DB_PASS; // e.g. 'my-db-password'
+  config.database = process.env.DB_NAME; // e.g. 'my-database'
+  config.server = process.env.DEPLOYED ? '172.17.0.1' : '127.0.0.1';
```

Comment:
Can we add a comment to this clarifying App Engine Flex uses `172.17.0.1`?

Reply:
done